src/components/link.js

```diff
@@ -6,6 +6,10 @@ import { Icon } from './icon'
 const ExternalLinkWrapper = styled.span`
   & > svg {
     opacity: 0.33;
+  };
```

Comment:
see comments below, but .. while this does work, i don't think adding functionality (and thus complexity) to this component is the cleanest approach